src/shell/dotnet/Shell/MainWindow.xaml.cs

```diff
@@ -51,6 +51,9 @@ private async void RibbonWindow_Initialized(object sender, System.EventArgs e)
         var modules = new List<ModuleViewModel>();
         foreach (var moduleId in moduleIds)
         {
+            // We dont want to see the chart example on the Ribbon menu, but without initializing in the module catalog we can't start via raiseIntent call
```

Comment:
Why don't we want to see the chart example on the Ribbon menu?

Reply:
Because it has no purpose. Right now the chart add its intent listener on load event. As right now the implementation of raiseIntent/addIntentListener could not popup a resolve UI to pick the right application(running instance), selecting a row from the Grid would just open a new Chart window even though we already opened the Chart from the Ribbon menu:
https://github.com/morganstanley/ComposeUI/blob/main/src/fdc3/js/composeui-fdc3/src/ComposeUIDesktopAgent.ts#L158

It picks up the first item from the response which should contain 2 elements. One is the Chart (not opened), and one is an instance of the chart.
As the first element would be the not running Chart- that can be opened, I do not see the purpose to set the Chart on the Ribbon menu, bacuse it wont do anything. :)

Reply:
Just as food for thought, as it really doesn't make much sense to open the Chart without any selected symbol, but for the sake of the example it might make sense to show that it can register an intent handler and listen to it even in that case.